templates/localvolumeprovisioner.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: localvolumeprovisioner
+  namespace: kubeaddons
+spec:
+  kubernetes:
+    minSupportedVersion: v1.14.0
+  chartReference:
+    chart: https://mesosphere.github.io/charts/stable/localvolumeprovisioner-0.1.0.tgz
+    # FIXME: Use the repo instead, once the index issues are resolved.
+    #chart: localvolumeprovisioner
```

Comment:
@gpaul We could revert this commented code now. It should work

Reply:
Done!